modules-js/react-fleet/src/utilities/constants.ts

```diff
@@ -0,0 +1,32 @@
+export const PUBLIC_CSS_URL = 'https://patterns.boston.gov/css/public.css';
+
+export const YELLOW = '#fcb61a';
+export const GREEN = '#62A744';
+export const GRAY_000 = '#f3f3f3';
+export const GRAY_100 = '#e0e0e0';
+export const GRAY_200 = '#C8C8C8';
+export const GRAY_300 = '#828282';
+export const OPTIMISTIC_BLUE = '#288be4';
+export const CHARLES_BLUE = '#091f2f';
+export const FREEDOM_RED = '#fb4d42';
+export const DEFAULT_TEXT = '#58585b';
+
+export const A11Y_RED = '#a32e26'; // higher contrast ratio; passes AAA
```

Comment:
Run this by Sebastian. I know that contrast ratio stuff has come up before, but I don't know the resolution.

+1 for paying attention to it, though.

Reply:
will do!